extension/content.js

```diff
@@ -71,6 +71,10 @@ function addReleasesTab() {
 function addTrendingMenuItem() {
 	const $secondListItem = $('.header-nav.left .header-nav-item:nth-child(2)');
 
+	if (pageDetect.isGist()) {
```

Comment:
Similar to the comments in #228, this doesn't follow the pattern for other page-specific implementations we have. I would take this logic out of the function and then wrap the function call in page detection, which lets this function only be responsible for the DOM updates.